src/ResourceManager/EventHub/Commands.EventHub/Utilities/EventHubsClient.cs

```diff
@@ -334,10 +399,32 @@ public void DeletConsumerGroup(string resourceGroupName, string namespaceName, s
             {
                 string test = ex.Message;
             }
-            
+
         }
 
 
         #endregion ConsumerGroup
+
+        #region Operations
+        public IEnumerable<PSOperationAttributes> GetOperations()
```

Comment:
Is this used? I can't find it anywhere.

Reply:
added the new cmdlet - Get-AzureRmEventHubOperation

Reply:
removed